planning/behavior_velocity_planner/include/utilization/interpolate.hpp

```diff
@@ -15,16 +15,22 @@
 #ifndef UTILIZATION__INTERPOLATE_HPP_
 #define UTILIZATION__INTERPOLATE_HPP_
 
+#include <rclcpp/rclcpp.hpp>
+
+#include <autoware_auto_planning_msgs/msg/path_point_with_lane_id.hpp>
+#include <autoware_auto_planning_msgs/msg/path_with_lane_id.hpp>
+
 #include <cmath>
 #include <vector>
 
 namespace behavior_velocity_planner
 {
+bool splineInterpolate(
+  const autoware_auto_planning_msgs::msg::PathWithLaneId & input, const double interval,
+  autoware_auto_planning_msgs::msg::PathWithLaneId * output, const rclcpp::Logger logger);
+
```

Comment:
Should it be in `interpolation` namespace as well as LinearInterpolate

Reply:
fixed [0f10e01](https://github.com/autowarefoundation/autoware.universe/pull/411/commits/0f10e0139fab838067bf714be05b85616365395e)